### ISSUE TITLE
TypeError: core.setWarn is not a function

### ISSUE THREAD

**Issue by garysb** [requarks/changelog-action#2]
I am receiving an error while trying to test the action:

```
(node:1584) UnhandledPromiseRejectionWarning: TypeError: core.setWarn is not a function
    at main (/home/runner/work/_actions/Requarks/changelog-action/v1.3.1/dist/index.js:26894:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1584) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:[15](https://github.com/area28technologies/gh-actions-test/runs/6321575423?check_suite_focus=true#step:3:15)84) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by NGPixel** [comment]
Fixed in 1.3.2

**Comment by garysb** [comment]
That was quick @NGPixel, thanks!
